packages/lib-react-components/src/Markdownz/Markdownz.js

```diff
@@ -60,12 +60,33 @@ class Markdownz extends React.Component {
       width = parseInt(match[1])
       height = parseInt(match[3])
       alt = alt.split(match[0])[0].trim()
+      if (width && !height) height = 'auto'
     }
 
     if (src) return <Media alt={alt} height={height} src={src} width={width} />
     return null
   }
 
+  replaceImageString (img) {
+    const imgSizeRegex = /=(\d+(%|px|em|rem|vw)?)x(\d+(%|px|em|rem|vh)?)?/
+    const imgSizeMatch = img.match(imgSizeRegex)
+
+    let newImgStr = ''
+
+    if (imgSizeMatch && imgSizeMatch.length > 0) {
+      // delete imgSizeMatch from alt and url string (including the preceding space character)
+      const sanitizedImgStr = img.replace(` ${imgSizeMatch[0]}`, '')
+
+      // place imgSizeMatch into only alt tag
+      const altTagRegex = /(?:!\[(.*?)\])/
+      const altTag = sanitizedImgStr.match(altTagRegex)
+      const altTextWithSize = `${altTag[1]} ${imgSizeMatch[0]}`
+
+      newImgStr = sanitizedImgStr.replace(altTag[0], `![${altTextWithSize}]`)
+    }
+    return newImgStr
+  }
```

Comment:
`imageRegex`, in the render method, includes regexes that capture the alt text, image URL and size. Those get passed as additional arguments to the substitution function, so you can rewrite this whole method as the following. That should remove the overhead of running some expensive regexes twice.
```suggestion
  replaceImageString (img, altText, imageURL, imageSize) {
    return `![${altText} ${imageSize}](${imageURL})`
  }
```